src/sentry/api/endpoints/organization_details.py

```diff
@@ -418,14 +522,15 @@ def put(self, request, organization):
                     data=changed_data,
                 )
 
-            return self.respond(
-                serialize(
-                    organization,
-                    request.user,
-                    org_serializers.DetailedOrganizationSerializerWithProjectsAndTeams(),
-                    access=request.access,
-                )
+            context = serialize(
+                organization,
+                request.user,
+                org_serializers.DetailedOrganizationSerializerWithProjectsAndTeams(),
+                access=request.access,
             )
+            self.append_trusted_relays_info(organization, context)
```

Comment:
Can you add this to the `context.update` block in [`DetailedOrganizationSerializer`](https://github.com/getsentry/sentry/blob/12f4ee73c414ef8472d390f017285868c8e97bc0/src/sentry/api/serializers/models/organization.py#L139) and restore this here? We generally try to have all such logic in serializers, which makes it easier to have an overview over what attributes are available in which scopes.

Reply:
done

Reply:
done